frontend/src/containers/LoginTopBar.js

```diff
@@ -20,6 +20,7 @@ export class LoginTopBar extends Component {
     return (
       <ul className='LoginTopBar-Links'>
         {this.showCreateBtn && <li><a className='LoginTopBarButton' href='/create'>create a collective</a></li>}
+        <li><a className='LoginTopBarLink' href='/learn'>How it works</a></li>
```

Comment:
there isn't a lot space between this and Discover on the page. Could we add a slight spacing. I like the spacing in the footer.